android/quest/src/main/java/org/smartregister/fhircore/quest/QuestApplication.kt

```diff
@@ -83,7 +86,13 @@ class QuestApplication : Application(), DataCaptureConfig.Provider, Configuratio
             enableAutoFragmentLifecycleTracing = true
           )
         )
+        try {
+          options.environment = URL(BuildConfig.FHIR_BASE_URL)?.getSubDomain()?.replace('-', '.')
+        } catch (e: Exception) {
+          Timber.e(e)
+        }
```

Comment:
Is this try-catch necessary?

Reply:
Was part of trying to troubleshoot which code updates introduced failing the Unit Test system

Reply:
Reverted